.buildkite/buildkite.yml

```diff
@@ -0,0 +1,44 @@
+projects:
```

Comment:
What beats are onboarded to this automated pipeline generator. 

These are folders from the top of the project. They should also host a similar `buildkite.yml` file.